cmd/move2kube/plan.go

```diff
@@ -125,7 +140,10 @@ func getPlanCommand() *cobra.Command {
 	planCmd.Flags().StringSliceVar(&flags.preSets, preSetFlag, []string{}, "Specify preset config to use")
 	planCmd.Flags().StringArrayVar(&flags.setconfigs, setConfigFlag, []string{}, "Specify config key-value pairs")
 
+	planCmd.Flags().IntVar(&flags.progressServerPort, "plan_progress_port", 0, "Port for the plan progress server. If not provided, the server won't be started.")
+
 	must(planCmd.MarkFlagRequired(sourceFlag))
+	must(planCmd.Flags().MarkHidden("plan_progress_port"))
```

Comment:
Can we move `plan_progress_port` as a flags file similar to our other flag names? Also, till we refactor other flag names to keep it consistent, lets use `planprogressport`. Its not ideal, but would be good to keep standard. Or we can refactor all flag names.